cupyx/scipy/fft/fft.py

```diff
@@ -6,7 +6,7 @@
 import cupy
 from cupy.cuda import cufft
 from cupy.fft.fft import (_fft, _default_fft_func, hfft as _hfft,
-                          ihfft as _ihfft)
+                          ihfft as _ihfft, _size_last_transform_axis)
 from cupy.fft.fft import fftshift, ifftshift, fftfreq, rfftfreq
```

Comment:
Sorry for my overlook before but importing non-module symbols is prohibited.

Reply:
@asi1024 Would you like me to fix all imports in this file, or just my addition?

Reply:
I think it was done like this for a reason: you can't import the module and then access the private functions without first exposing them in `cupy/fft/fft/__init__.py`...

Reply:
Any comment, @asi1024?

Reply:
Ah, I see. Thanks!